integration/dbutils/migrator.spec.ts

```diff
@@ -12,6 +12,14 @@ var dbShape : Shapes.DBShape = {
   tables: [{
     tableName: 'users',
     indices: ['userName', 'email']
+  },
+  {
+    tableName: 'userData',
+    indices: []
+  },
+  {
+    tableName: 'activation',
```

Comment:
Ah, I see your point. But in the future when we do use the `userData` table, won't the queries be just as complex? How did you plan to store data in the `userData` table?

Would it be easier if the `userData` table just had two columns, one being the user's ID and the other being values, where values is just a JSON object?